js/lexicon/lexicon-index.js

```diff
@@ -0,0 +1 @@
+// Something is going to be added here soon
```

Comment:
Could you also add the bundled lexicon javascript to the lexicon templates, it might be nice to stick a console log in there so you can test that this file is included in lexicon, and has access to the `Pulsar` global variable.